tests/sqlparser_snowflake.rs

```diff
@@ -4165,3 +4167,10 @@ fn test_snowflake_fetch_clause_syntax() {
         canonical,
     );
 }
+
+#[test]
+fn test_snowflake_create_view_with_tag() {
+    let create_view_with_tag =
+        r#"CREATE VIEW X (COL WITH TAG (pii='email') COMMENT 'foobar') AS SELECT * FROM Y"#;
```

Comment:
is the intent to verify the behavior of tag or multiple comments support? my understanding is the latter so that we could probably rename the test to match?